src/ResourceManager/Automation/Commands.Automation.Test/UnitTests/GetAzureAutomationHybridWorkerGroupTest.cs

```diff
@@ -0,0 +1,75 @@
+﻿using Microsoft.Azure.Commands.Automation.Cmdlet;
+using Microsoft.Azure.Commands.Automation.Common;
+using Microsoft.Azure.Commands.Automation.Model;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using Microsoft.WindowsAzure.Commands.Common.Test.Mocks;
+using Microsoft.WindowsAzure.Commands.Test.Utilities.Common;
+using Microsoft.WindowsAzure.Commands.Utilities.Common;
+using Moq;
+using System.Collections.Generic;
+
+namespace Microsoft.Azure.Commands.ResourceManager.Automation.Test.UnitTests
+{
+    [TestClass]
+    public class GetAzureAutomationHybridWorkerGroupTest : RMTestBase
+    {
+        private Mock<IAutomationClient> mockAutomationClient;
+
+        private MockCommandRuntime mockCommandRuntime;
+
+        private GetAzureAutomationHybridWorkerGroup cmdlet;
+
+        [TestInitialize]
+        public void SetupTest()
+        {
+            this.mockAutomationClient = new Mock<IAutomationClient>();
```

Comment:
In Xunit, this can just be in the class constructor, as long as it can run before each test.  You can do suite-based intiailization using TestFixtures